src/boot/bootctl.c

```diff
@@ -123,12 +123,8 @@ static int verify_esp(
 
         errno = 0;
         b = blkid_new_probe_from_filename(t);
-        if (!b) {
-                if (errno == 0)
-                        return log_oom();
-
-                return log_error_errno(errno, "Failed to open file system \"%s\": %m", p);
-        }
+        if (!b)
+                return log_error_errno(errno ?: ENOMEM, "Failed to open file system \"%s\": %m", p);
```

Comment:
still don't like it if we log errno constants we are making up instead of those we receive...